hot-reload-analysis/src/main/kotlin/org/jetbrains/compose/reload/analysis/RuntimeInstructionTree.kt

```diff
@@ -254,3 +251,168 @@ private fun parseRuntimeInstructionTree(
         lastIndex = index
     ).toLeft()
 }
+
+
+private class MutableTree(
+    var group: ComposeGroupKey?,
+    var type: RuntimeScopeType,
+    val tokens: MutableList<RuntimeInstructionToken>,
+    val children: MutableList<MutableTree>,
+    var startIndex: Int,
+    var endIndex: Int,
```

Comment:
In the new algorithm, do we actually care about the start and endIndex? Maybe we can just drop it from the tree completely?